parsec/test_utils/organization.py

```diff
@@ -136,51 +138,51 @@ async def initialize_test_organization(
     return (alice_device, other_alice_device, bob_device)
 
 
+async def _add_random_device(cmds, config_dir, password, force, device, additional_devices_number):
+    for _ in range(additional_devices_number):
+        device_label = "device_" + str(uuid4())[:9]
+        new_device = await _register_new_device(cmds=cmds, author=device, device_label=device_label)
+        save_device_with_password(config_dir, new_device, password, force=force)
```

Comment:
do we really need to save the devices here ? I would say no given we don't save the users added from `_add_random_users`